pkgs/applications/video/gnomecast/default.nix

```diff
@@ -17,7 +17,10 @@ buildPythonApplication rec {
   ];
 
   preFixup = ''
-    gappsWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ ffmpeg_3 ]})
+    gappsWrapperArgs+=( \
+      --set GDK_PIXBUF_MODULE_FILE ${librsvg.out}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache \
```

Comment:
@andresilva why is the `.out` needed?

Reply:
It isn't, good catch! I think I just copy pasted this line from elsewhere while trying to figure out the issue. Pushed a commit to remove the `.out`.